src/cli/db-migration.ts

```diff
@@ -0,0 +1,154 @@
+import fs from 'fs'
```

Comment:
This command should replace the `db-sync` command. Not sure if this command should be renamed to `db-sync` for backward compatibility or just remove the `db-sync` and keep  `db-migrations` 🤔 I will leave that to your decision.

Reply:
Ok. Probably I remove db-sync, and leave migration command as it is